tests/components/heos/conftest.py

```diff
@@ -82,7 +82,36 @@ def player_fixture(quick_selects):
     player.now_playing_media.image_url = "http://"
     player.now_playing_media.song = "Song"
     player.get_quick_selects.return_value = quick_selects
-    return {player.player_id: player}
+
+    player2 = Mock(HeosPlayer)
```

Comment:
Revert all of the `player2` changes as it's no longer necessary to test the changes of setting `unique_id`.